SEED-platform/seed
# Update portfolio summary

#### Any background context you want to provide?
User requested updates

#### What's this PR do?
- Adds new `Goal.commitment_sqft` field
- Adds total properties and commitment to goal details gray box
- Adds a goal stats table with Commitment Sq Ft, Shared Sq Ft, Passing Checks Sq Ft, Passing Checks % of Committed, Passing Checks % of Shared, Total Passing Checks, Total New or Acquired
- Added ui-grid actions dropdown featuring ability to Select All, Select None, Add/Remove Labels, Bulk Edit Goal Notes, Show/Hide Access Levels
- New Modal for bulk editing `GoalNotes`
- Creates `GoalNotes` for newly imported properties

#### How should this be manually tested?
Confirm the following
- Commitment Sq Ft = user input
- Shared Sq Ft = total sqft 
- Passing Checks Sq Ft = total sqft of properties with passing checks
- Percentages are accurate
- Totals are accurate
- Portfolio Summary values are only calculated with properties passing checks and not new or acquires
- Actions - Add/Remove Labels
- Actions - Bulk edit selected goal notes (blank is a valid input as long as the field is selected)

#### What are the relevant tickets?
#4661

#### Screenshots (if appropriate)
![Screenshot 2024-05-10 at 2 15 22 PM](https://github.com/SEED-platform/seed/assets/58446472/e48b04eb-c204-4b46-b135-83d07316609a)

![Screenshot 2024-05-10 at 2 15 30 PM](https://github.com/SEED-platform/seed/assets/58446472/b9c5a86f-58b8-4a80-b69a-3dc3d03cebd9)
![Screenshot 2024-05-10 at 2 17 11 PM](https://github.com/SEED-platform/seed/assets/58446472/21cc0e86-2431-48a4-b182-5b3bacf9e812)
